src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.hxx

```diff
@@ -74,6 +74,14 @@ class t8_geometry_squared_disk: public t8_geometry {
     SC_ABORT_NOT_REACHED ();
   }
 
+  void
+  t8_geom_point_batch_inside_element (t8_forest_t forest, t8_locidx_t ltreeid, const t8_element_t *element,
```

Comment:
Why repeat this routine here. It derives from `geometry_with_vertices` where this routine is already implemented as virtual function.